src/Tribe/REST/V1/Post_Repository.php

```diff
@@ -76,6 +76,7 @@ public function get_event_data( $event_id, $context = '' ) {
 
 		$data = array(
 			'id'                     => $event_id,
+			'checksum'               => tribe_post_checksum( $event_id ),
```

Comment:
Why not pass in $event here to avoid additional get_post call?

Reply:
Good catch, thanks.